core/text/table/table.odin

```diff
@@ -165,7 +180,7 @@ first_row :: proc(tbl: ^Table) -> int {
 	return header_row(tbl)+1 if tbl.has_header_row else 0
 }
 
-build :: proc(tbl: ^Table) {
+build :: proc(tbl: ^Table, width_proc: Width_Proc) {
```

Comment:
I think I would prefer a default Width_Proc here. Default to `unicode_width_proc`.

But considering previous comment about ascii being a subset, is Width_Proc needed at all? Any use case for custom width procs you can think of?

Reply:
Given your comment about `grapheme_count` I change my mind here. Leave it as it is.